hi-ml-histopathology/testhisto/testhisto/models/test_deepmil.py

```diff
@@ -129,26 +131,34 @@ def _test_lightningmodule(
 
 
 @pytest.fixture(scope="session")
-def mock_panda_tiles_root_dir(tmp_path_factory: pytest.TempPathFactory) -> Path:
+def mock_panda_tiles_root_dir(
+    tmp_path_factory: pytest.TempPathFactory, tmp_path_to_pathmnist_dataset: Path
+) -> Generator:
     tmp_root_dir = tmp_path_factory.mktemp("mock_tiles")
     tiles_generator = MockPandaTilesGenerator(
-        tmp_path=tmp_root_dir,
+        dest_data_path=tmp_root_dir,
+        src_data_path=tmp_path_to_pathmnist_dataset,
         mock_type=MockHistoDataType.PATHMNIST,
         n_tiles=4,
         n_slides=10,
         n_channels=3,
         tile_size=28,
         img_size=224,
     )
+    logging.info("Generating mock tiles")
```

Comment:
It is valuable to say here that these mock tiles will be deleted at the end of the session